src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -133,8 +136,8 @@ public void stopCDSWheel() {
         ballCount++;
       }
     }
-
     SmartDashboard.putNumber("Ball Count", ballCount);
+    Shuffleboard.addEventMarker("" + ballCount, EventImportance.kNormal);
```

Comment:
Let's remove this, as we already log the ball count.  Should the ball count be moved to operator or driver tab? That would be a good change here.

Reply:
It has been deleted in the latest version of main and moved to only activate when debug mode is enabled.